easybuild/easyconfigs/i/icc/icc-2016.0.109.eb

```diff
@@ -0,0 +1,23 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
```

Comment:
Shouldn't we add a comment that we implicietly use the system GCC here? To clearly state the difference with the previous easyconfig for new people? I would take this one if I didn't know better :wink: